src/torchcodec/_core/CudaDevice.cpp

```diff
@@ -217,40 +218,41 @@ void convertAVFrameToFrameOutputOnCuda(
         "x3, got ",
         shape);
   } else {
-    dst = allocateEmptyHWCTensor(height, width, videoStreamOptions.device);
+    dst =
+        allocate_empty_h_w_c_tensor(height, width, video_stream_options.device);
```

Comment:
Here and below, I'd find it easier if we wrote this as `allocate_empty_hwc_tensor()` and `nppi_nv12_to_rgb_8u_p2c3r()`. Basically, we'll want to be careful about how to turn a sequence of all caps into snake case.